assets/js/components/ExecutionResults/CheckResultDetail/CheckResultDetail.stories.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  addPassingExpectExpectation,
+  addPassingExpectSameExpectation,
+  emptyCheckResultFactory,
+  hostFactory,
+  checksExecutionCompletedFactory,
+  checkResultFactory,
+  addCriticalExpectExpectation,
+} from '@lib/test-utils/factories';
+
+import CheckResultDetail from './CheckResultDetail';
+
+const clusterHosts = hostFactory.buildList(2);
+const [{ id: target1 }, { id: target2 }] = clusterHosts;
+const targetType = 'host';
+
+let checkResult = emptyCheckResultFactory.build({
+  targets: [target1, target2],
+});
+
+const { check_id: checkID } = checkResult;
+
+checkResult = addPassingExpectExpectation(checkResult);
+checkResult = addPassingExpectExpectation(checkResult);
+checkResult = addCriticalExpectExpectation(checkResult);
+checkResult = addCriticalExpectExpectation(checkResult);
+checkResult = addCriticalExpectExpectation(checkResult);
+checkResult = addPassingExpectSameExpectation(checkResult, 'expectation_name');
+
+const executionData = checksExecutionCompletedFactory.build({
+  check_results: [checkResultFactory.build(), checkResult],
+});
+
+export default {
+  title: 'CheckResultDetail',
+  component: CheckResultDetail,
+};
+
+export function Default() {
+  return (
+    <CheckResultDetail
+      checkID={checkID}
+      targetID={target1}
+      targetType={targetType}
+      executionData={executionData}
+      clusterHosts={clusterHosts}
```

Comment:
`clusterHosts` is not a `CheckResultDetail` prop. In fact, the `expectations` prop is missing here.
It doesn't really affect the current view, as the `expectations` are only used to get `expectSame` kind of info.

Reply:
You're absolutely right, thank you for the heads up :+1: